src/guides/v2.4/graphql/queries/pickup-locations.md

```diff
@@ -105,6 +106,23 @@ Because `pageSize` is set to 1 and distance by `distance` is applied, result wil
   }
 }
 ```
+If the Pickup location is not available for the given search term, the response will be empty:
+
+```
+{
+  "data": {
+    "pickupLocations": {
+      "items": [],
+      "total_count": 0,
+      "page_info": {
+        "page_size": 1,
+        "current_page": 1,
+        "total_pages": 0
+      }
+    }
+  }
+}
+```
```

Comment:
We don't need to show an empty response. Remove all of your additions up to here.